scripts/hurricane/stats/exevs_hurricane_regional_tropcyc_stats.sh

```diff
@@ -208,7 +231,7 @@ cd $metTCcomout
 #export SEARCH3=INIT_BEG_template
 #export SEARCH4=INIT_END_template
 
-cp ${PARMevs}/metplus_config/hurricane/stats/TCStat_template_basin.conf .
+cp ${PARMevs}/TCStat_template_basin.conf .
 
```

Comment:
You will want to get this change back into your PR. You accidentally introduced older code for line 211.